crawler/attribution/entrypoints/data-url.js

```diff
@@ -0,0 +1,7 @@
+{
+    const u = `data:application/javascript,console.log("${document.currentScript.src}", navigator.userAgent)`;
```

Comment:
The result here isn't being used, shouldn't we be calling addResult from inside data?

Reply:
Good catch, thanks!